thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/autoload/pinot/metrics/AutoloadPinotMetricsServiceTest.java

```diff
@@ -36,6 +37,17 @@ void afterClass() {
     super.cleanup();
   }
 
+  /**
+   * This test class needs to use a class that accesses the global singleton DAO registry. Therefore,
+   * we need to copy the local DAO registry, which is created for this test, to the global one.
+   * Otherwise, the global singleton one may reference to an arbitrary DB that is created for this
+   * test.
+   */
+  @BeforeMethod
+  void beforeMethod(){
```

Comment:
Don't we need to do this for all the other tests (TestAnomalyFunctionManager, TestAlertCOnfigManager etc)?